Jenkinsfile

```diff
@@ -58,6 +59,32 @@ void unittests(String version) {
     }
 }
 
+void uploadPackagePyPI() {
+    node(Constants.DOCKERNODE) {
+        buildsCommon.cleanup()
+        checkout scm
+        withContainer(image: "python:3.6-buster", registry: '', inside: '-u root') {
+            try {
+                if (env.BRANCH_NAME == 'master') {
```

Comment:
We have two if ..... that should be combine.
Why do spin up an EC2 executor, container to then only execute the logic on the master branch.

Wasteful an expensive.

Reply:
As only with that, we might execute the setup.py, so we need to do this